<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATCommandPage</class>
 <widget class="QWidget" name="ATCommandPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AT Command Editor</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <!-- 主编辑区 -->
   <item>
    <layout class="QHBoxLayout" name="contentLayout">
     <!-- 中间编辑区 -->
     <item>
      <widget class="QTextEdit" name="commandTextEdit">
       <property name="objectName">
        <string>text_input_layout_2</string>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::WidgetWidth</enum>
       </property>
      </widget>
     </item>
     <!-- 右侧路径选择区 -->
     <item>
      <widget class="QScrollArea" name="pathScrollArea">
       <property name="objectName">
        <string>pathScrollArea</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
        </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <widget class="QGroupBox" name="pathGroupBox">
        <property name="title">
         <string>AT Command Paths</string>
        </property>
        <layout class="QGridLayout" name="pathGridLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
